lib/image-stream/handlers.js

```diff
@@ -50,6 +52,8 @@ module.exports = {
   'application/x-bzip2': (file, options) => {
     return Bluebird.props({
       path: file,
+      archiveExtension: path.extname(file),
```

Comment:
I think that the archive extension is redundant, given that it can be obtained from `path`. Can we only add `extension`?